condor_watch_q.py

```diff
@@ -195,6 +195,7 @@ def cli():
 
 
 def watch_q(
+## MAIN TABLE PRESENTATION
```

Comment:
I generally dislike comments like this. If necessary, they should be function-level docstrings like https://github.com/htcondor/htmap/blob/master/htmap/maps.py#L115 . If a subsection of code can be identified and needs its own name, let's pull it out into a new function with that name.

Please remove all such comments that you added.